lib/Transforms/Scalar/ScalarReplAggregatesHLSL.cpp

```diff
@@ -3531,15 +3554,19 @@ static bool ReplaceMemcpy(Value *V, Value *Src, MemCpyInst *MC,
       } else {
         // Replace Constant with a non-Constant.
         IRBuilder<> Builder(MC);
-        ReplaceConstantWithInst(C, Src, Builder);
+        if (!ReplaceConstantWithInst(C, Src, Builder)) {
+          return false;
+        }
```

Comment:
nit:
```suggestion
        if (!ReplaceConstantWithInst(C, Src, Builder))
          return false;
```
see: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements